sphinx_doc/build_instructions/build.rst

```diff
@@ -0,0 +1,235 @@
+Build Instructions
+##################
+
+Overview
+********
+
+The instructions to build **Navigation2 Stack** and **Navigation2 Dependencies**.
+
+- Build Options
+  
+  - `Quickstart Using Initial Setup Script`_
+  - `Manually Build From Source`_
+  
+- `Build Troubleshooting Guide`_
+
+**Recommended Platform**
+
+* `Ubuntu 18.04 <http://releases.ubuntu.com/18.04/ubuntu-18.04.3-desktop-amd64.iso>`_
+
+**Build Options**
+
+There two different recommended methods for building Navigation2 stack from source.
+
+**Easy Build**
+
+* An automated method to build Navigation2 in a simple and easy way.
+* To build Navigation2 by using the build script, go to `Quickstart using initial setup script`_
+
+Manually Build From Source
+**************************
+
+Step 1- Build ROS 2
+===================
+
+* `ROS2 Build Instructions <https://index.ros.org/doc/ros2/Installation>`_
+* When building ROS 2 from source, if you want to get the latest version of the ROS 2 packages, make sure that the `ros2.repos` file is from the `master` branch.
+* When installing the ROS 2 dependencies using rosdep, make sure to set `--rosdistro` to the lastest ROS 2 distribution (e.g `eloquent`)
+
+Step 2- Build Navigation2 Dependencies
+======================================
+
+- If you built ROS 2 from source, source the setup.bash file in the ROS 2 build workspace.
+  
+    ``source ~/ros2_ws/install/setup.bash``
+
+- If you installed ROS 2 from the binary packages, source the setup.bash file in the ROS 2 installation directory.
+    
+    ``source /opt/ros/<ros2-distro>/setup.bash``
+
+**Build and install Navigation2 dependencies:**
+
+.. code:: bash
+
+  mkdir -p ~/nav2_depend_ws/src
+  cd ~/nav2_depend_ws
+  wget https://raw.githubusercontent.com/ros-planning/navigation2/master/tools/ros2_dependencies.repos
+  vcs import src < ros2_dependencies.repos
+  rosdep install -y -r -q --from-paths src --ignore-src --rosdistro eloquent
+  colcon build --symlink-install
+
+
+Step 3- Build Navigation2
+=========================
+
+**I want to build the latest version:**
```

Comment:
I want to build master branch, latest version is really unclear, I would think the latest version is the latest released distribution, not the master non-named distribution branch